resources/sdk/clisdkclient/extensions/restclient/tls/generate_cert.go

```diff
@@ -39,15 +39,10 @@ func GenerateCerts() error {
 		err            error
 		pathToCertFile string
 		pathToKeyFile  string
-		tmpDir         = os.TempDir()
 	)
 
-	if !strings.HasSuffix(tmpDir, "/") {
-		tmpDir += "/"
-	}
-
-	pathToCertFile = tmpDir + certFileName
-	pathToKeyFile = tmpDir + keyFileName
+	pathToCertFile = filepath.Join(os.TempDir(), certFileName)
+	pathToKeyFile = filepath.Join(os.TempDir(), keyFileName)
```

Comment:
This is the change that fixed the issue. I should have been using `filepath.Join()` instead of manually adding a "/", because in windows it should be a "\" , so the server couldn't access the key/cert files because the path was invalid. Not sure how this code used to work for windows and stopped working all the sudden, but it's fixed now anyway